commands/command_unlock.go

```diff
@@ -31,11 +33,16 @@ func unlockCommand(cmd *cobra.Command, args []string) {
 			Exit("Unable to determine path: %v", err.Error())
 		}
 
+		unlockCheckFileStatus(path)
```

Comment:
Yeah I originally had this function returning but it meant duplicating the outcomes (it's needed in a derived form when checking by Id), and also duplicating the override of that with `--force`. We have form in the `commands` package of `Exit`ing inside functions so I thought this was the simplest in the round. I'll comment it though and maybe change the name to `unlockAbortIfFileUncommitted`.